test/test_literal.py

```diff
@@ -163,6 +186,7 @@ def unlexify(s):
         self.assertEqual(specific_l.toPython(), s)
         self.assertEqual(specific_l.datatype, datatype)
 
+ 
```

Comment:
Could you please remove this white space line. The rest is cool.

Reply:
Removed the whitespace line. Please let me know if any other changes are needed.